src/main.js

```diff
@@ -35,6 +35,24 @@ let load = base.loader({
     }),
   },
 
+  docs: {
+    requires: ['cfg', 'validator', 'reference'],
+    setup: ({cfg, validator, reference}) => base.docs({
+      credentials: cfg.taskcluster.credentials,
```

Comment:
project name is automatically grabbed from package.json unless otherwise specified.

menuIndex is set to 10 unless otherwise specified. I figure the index will be sorted by

1. menuIndex
2. alphabetically

So that things are stable when we don't care about the ordering.